handbook/engineering/distribution/index.md

```diff
@@ -4,11 +4,15 @@
 
 The distribution team is responsible for making Sourcegraph easy to deploy, scale, monitor, and debug. We solve the most challenging problems our customers face deploying Sourcegraph on-premise in a variety of environments, and on [Sourcegraph.com](https://sourcegraph.com/search) - the largest Sourcegraph instance in the world.
 
-## Contact
+## Communication
 
 - Slack: #distributioneers channel or @distribution
 - File issues: [team/distribution](https://github.com/sourcegraph/sourcegraph/issues/new?labels=team/distribution) label
 
+Our team has two Slack channels, one public (#distributioneers) and one private (#distribution-team). **Our default is to use the public channel**.
+
+The #distribution-team channel will be used for discussing internal team topics, like scheduling, retrospectives, how we do updates, etc.
```

Comment:
We should have a "Teams communication" section instead, like Campaigns does (they also reference a private channel): https://about.sourcegraph.com/handbook/engineering/campaigns#team-communication

Reply:
Ill move it. Just for reference, I thought it would be good to have this at the top and in the same section given its just a small paragraph. This is also the reasoning behind the `Contact -> Communication`.